index.js

```diff
@@ -1,118 +1,235 @@
 
+let ageInDogYears;
+let humanAge;
+let ageConverter = function(humanAge) {
+    ageInDogYears = humanAge * 7;
+    console.log("I am " + ageInDogYears + " in dog years.");
+    }
+    ageConverter(32);
+// returning undefined for some reason... grrr
 
 /************************************************************** Task 3 **************************************************************/
-//Dog feeder 
+//Dog feeder
 //takes weight in pounds and age in years (note if the dog is a puppy the age will be a decimal) and returns the number of pounds of raw food to feed in a day.
 
 //feeding requirements
-// adult dogs at least 1 year 
+// adult dogs at least 1 year
 // up to 5 lbs - 5% of their body weight
-// 6 - 10 lbs - 4% of their body weight 
-// 11 - 15 lbs - 3% of their body weight 
-// > 15lbs - 2% of their body weight 
+// 6 - 10 lbs - 4% of their body weight
+// 11 - 15 lbs - 3% of their body weight
+// > 15lbs - 2% of their body weight
 
 // Puppies less than 1 year
 // 2 - 4 months 10% of their body weight
-// 4 - 7 months 5% of their body weight 
+// 4 - 7 months 5% of their body weight
 // 7 - 12 months 4% of their body weight
 
 // when you are finished invoke your function with the weight of 15 lbs and the age of 1 year - if your calculations are correct your result should be 0.44999999999999996
-  
-
 
 
+let dogFeeder = function (age, weight){
+    if( age >= 1 ) {
+        if( weight <= 5 ) {
+            feed = weight * .05;
+        } else if ( 5 < weight && weight <= 10 ) {
+            feed = weight * .04;
+        } else if ( 10 < weight && weight <= 15 ) {
+            feed = weight * .03;
+        } else if ( weight < 15 ) {
+            feed = weight * .02;
+        // } else {
+        //     console.log("Please try again with dog's weight")
+        }
+    } else if ( (2/12) < age && age < (4/12) ) {
+        feed = weight * .1;
+    } else if ( (4/12) <= age && age < (7/12) ) {
+        feed = weight * .05;
+    } else if ( (7/12) <= age && age < 1 ) {
+        feed = weight * .04;
+    }
+    console.log("Feed Me " + feed + " pounds of raw food a day");
+}
+dogFeeder(1, 15);
 
 /************************************************************** Task 4 **************************************************************/
 // Rock, Paper, Sissors
 // Your function should take a string (either rock paper or sissors)
 // it should return you won or you lost based on the rules of the game (you may need to look up the rules if you have not played before)
-// use math.random to determine the computers choice 
-// hint while you can complete this with only conditionals based on strings it may help to equate choice to a number 
+// use math.random to determine the computers choice
+// hint while you can complete this with only conditionals based on strings it may help to equate choice to a number
+
+
+// sytax expanded bc it was frying my brain...
+// 99% sure there's a more elegant solution in here. Excited to find it later.
```

Comment:
There is always a more elegant solution. There will usually be various trade-offs between them. Here, you've fully expanded the logic, which is much easier to follow manually but leaves more room for errors. Others might be shorter or more elegant or optimized, but be more difficult to understand easily.